ServiceWebsite/ServiceWebsite.Services/HearingsService.cs

```diff
@@ -43,7 +43,36 @@ private static bool HearingContainsParticipant(HearingDetailsResponse response,
         {
             return response.Participants.Any(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
         }
-        
+
+        public async Task<Guid?> GetParticipantIdAsync(string username, Guid hearingId)
+        {
+            try
+            {
+                var hearingResponse = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
+                var participant = hearingResponse.Participants.FirstOrDefault(p => p.Username.Equals(username, StringComparison.OrdinalIgnoreCase));
+
+                if(participant==null)
+                {
+                    return Guid.Empty;
+                }
+
+                var participantId = participant.Id;
```

Comment:
Again, you can return this directly, no need to store it in a variable:

```csharp
return participant.Id;
```

Reply:
Done